lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -113,7 +156,7 @@ async def _get_subscribers(self) -> [SubscriberData]:
                     time_elapsed.total_seconds() * 1000,
                 )
                 SUBSCRIBER_SYNC_FAILURE_TOTAL.inc()
-                return None
+                return None, None
         logging.info(
```

Comment:
For tuple structs like this I would recommend using a NamedTuple and explicitly name the parameters instead of returning a tuple here, it doesn't have to be in this PR you can add it as a TODO and add this later.

Reply:
sounds good, will do!